src/views/bundler/Bundler.tsx

```diff
@@ -90,7 +99,77 @@ function Bundler(props: any) {
         setRows(rows);
         setTableLoading(false);
     };
+ 
+    // const checkTermInNetworks = async (term) => {
```

Comment:
Do we need this commented code ?

Reply:
removed in  [cf8bf85](https://github.com/jiffy-labs/jiffyscan-frontend/pull/129/commits/cf8bf85bc6ecfbfc23f5300811992a8eb84a22de)